src/panels/Scene/SceneTree/SceneTree.tsx

```diff
@@ -103,8 +107,21 @@ export function SceneTree() {
     return nameA.toLocaleLowerCase().localeCompare(nameB.toLocaleLowerCase());
   });
 
+  function computeHeight(windowHeight: number): number {
```

Comment:
This seems like a completely copied version of the function in OriginPanel? Can we generalize it and put it in some utility function instead?